docs/crio.8.md

```diff
@@ -89,7 +89,7 @@ crio [GLOBAL OPTIONS] config [OPTIONS]
 
 **--config**="": path to configuration file
 
-**--conmon**="": path to the conmon executable (default: "/usr/local/libexec/crio/conmon")
+**--conmon**="": Path to the conmon binary, used for monitoring the OCI runtime. Will be searched inside $PATH if empty. (default: "")
```

Comment:
suggest: "searched inside" to "searched for using"

Reply:
Yes, I changed it as suggested. :+1: